java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Request/PersonRequest.java

```diff
@@ -17,4 +17,6 @@
   private Confidence confidence;
   @Builder.Default private boolean infer = true;
   private List<String> dataFilter;
+  private boolean verifiedPhysical, expandedInterests;
+  private Integer maxMaids;
```

Comment:
Please add javadoc documentation what these fields for.

Reply:
Done